Tests/KituraTests/KituraTest.swift

```diff
@@ -61,6 +61,16 @@ class KituraTest: XCTestCase {
     override func setUp() {
         super.setUp()
         KituraTest.initOnce
+#if USE_EPHEMERAL_PORTS
+        func shell(_ args: String...) {
+            let task = Process()
+            task.launchPath = "/usr/bin/env"
+            task.arguments = args
+            task.launch()
+            task.waitUntilExit()
+        }
+        shell("./Tests/KituraTests/setup.sh")
+#endif
```

Comment:
Shouldn't this be in the singleton init (if we're going to run it from the tests at all)?

I would like to understand why we would want to do this at all though. Shouldn't Travis just run those commands before running the tests? If it's to make local testing easier, then I think it's reasonable to ask the user to do this if it's necessary.

Reply:
Yes, it is for local testing. I didn't want to break the usual `clone-build-test` sequence. We could do away with this by providing instructions for local testing, but again, there's is no easily accessible section in the README to add this.